Client/Frontend/Browser/URLBarView.swift

```diff
@@ -69,6 +69,10 @@ class URLBarView: UIView {
         return locationView
     }()
 
+    func textFieldIsFirstResponder() -> Bool {
+        return locationTextField.isFirstResponder()
+    }
+
```

Comment:
Hopefully we won't need this (see [comment](https://github.com/mozilla/firefox-ios/pull/845/files#r36241592)).